src/main/java/com/microsoft/sqlserver/jdbc/IOBuffer.java

```diff
@@ -2346,21 +2463,22 @@ Socket findSocket(String hostName, int portNumber, int timeoutInMilliSeconds, bo
                 logger.finer(loggingString.toString());
             }
 
-            if (inetAddrs.length > ipAddressLimit) {
+            if (inetAddrs.length > IPADDRESSLIMIT) {
                 MessageFormat form = new MessageFormat(
-                        SQLServerException.getErrString("R_ipAddressLimitWithMultiSubnetFailover"));
-                Object[] msgArgs = {Integer.toString(ipAddressLimit)};
+                        SQLServerException.getErrString("R_IPADDRESSLIMITWithMultiSubnetFailover"));
```

Comment:
did you ctrl+f replace the ipAddressLimit to IPADDRESSLIMIT instead of refactoring?